codecparsers/h265Parser.cpp

```diff
@@ -24,6 +24,38 @@
 #include <math.h>
 #include <stddef.h> //offsetof
 
+#define READ(x, f)                          \
```

Comment:
I think this one is much more clearer, those who read the code will clearly know whether it is br or nr they want to use. When I previously read the code of h264 I am confuse at why we need to declare a nr with name br. With this change, I think the code is more easily to read.